gitlab/client.py

```diff
@@ -269,7 +269,7 @@ def version(self) -> Tuple[str, str]:
         object.
 
         Returns:
-            tuple (str, str): The server version and server revision.
+            tuple: The server version and server revision.
                               ('unknown', 'unknwown') if the server doesn't
```

Comment:
While in here, maybe do:  s/unknwown/unknown/  to fix the mis-spelling.

Reply:
Oh, thanks. Even the `tuple` here is not needed 🤔  seems like my search/replace missed a few of these, I'll check again.

Reply:
https://github.com/python-gitlab/python-gitlab/pull/1709/commits/79e785e765f4219fe6001ef7044235b82c5e7754 also cleans up return types and a bit of formatting.